code/src/main/java/com/onlinecontacttracing/servlets/get-data/CalendarDataForPositiveUser.java

```diff
@@ -1,16 +1,74 @@
 package com.onlinecontacttracing.authentication;
 
 import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.Events;
+import com.onlinecontacttracing.storage.Constants;
+import com.onlinecontacttracing.storage.PositiveUserContacts;
+import com.google.api.services.calendar.model.EventAttendee;
+import java.util.List;
+import java.util.Collections;
+import java.util.Optional;
+import com.googlecode.objectify.Objectify;
 
 class CalendarDataForPositiveUser implements Runnable {
+
+  private static final String APPLICATION_NAME = "Google Calendar API Java Quickstart";
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+  private final String userId;
   private final Credential credential;
+  private final Objectify ofy;
 
-  public CalendarDataForPositiveUser(Credential credential) {
+  public CalendarDataForPositiveUser(Objectify ofy, String userId, Credential credential) {
+    this.userId = userId;
     this.credential = credential;
+    this.ofy = ofy;
   }
 
   @Override
   public void run() {
-    // TODO Get contacts from people api
+    try {
+      final NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
```

Comment:
why does this variable need to be final?

Also, since this isn't static + final, please make it lowerCamelCaseVariableName.

UPPER_SNAKE is reserved for static final vars.

Reply:
This is how we have it in all the code

Reply:
It came from the quickstart like this

Reply:
OK - well then feel free to fix it everywhere :)

Reply:
I changed it to camel case but I won't make it non final since I am not sure what it does and wouldn't want to mess it up